corehq/motech/repeaters/static/repeaters/js/bootstrap5/repeat_record_report.js

```diff
@@ -273,22 +275,41 @@ hqDefine('repeaters/js/bootstrap5/repeat_record_report', [
         // ----- what was once repeat_record_report_selects.js ------
 
         $('#select-all-checkbox').on('click', function () {
-            toggleItems(selectAll.checked);
-            uncheckSelects();
+            if (selectAllCheckbox.checked) {
+                toggleItems(true);
+                selectedPageInfo.classList.remove('hide');
```

Comment:
B5 doesn't have the `hide` class anymore, you can use `d-none` instead.

Reply:
00533dda42639e64ec8ec5dee67ba8b56f8173cc